examples/howto/plot_batch_simulate.py

```diff
@@ -111,18 +124,47 @@ def summary_func(results):
 print("Simulation results:", simulation_results)
 ###############################################################################
 # This plot shows an overlay of all smoothed dipole waveforms from the
-# batch simulation. Each line represents a different set of parameters,
-# allowing us to visualize the range of responses across the parameter space.
-
-dpl_waveforms = []
+# batch simulation. Each line represents a different set of synaptic strength
+# parameters (`weight_basket`), allowing us to visualize the range of responses
+# across the parameter space.
+# The colormap represents different synaptic strengths, with purple indicating
+# lower strengths and yellow indicating higher strengths.
+#
+# Key observations:
+#
+# - The dipole signal reflects the net current flow in the cortical column.
+# - Initially, we see a positive deflection as excitatory input arrives at
+#   the proximal dendrites, causing current to flow upwards
+#   (away from the soma).
+# - The subsequent negative deflection, despite continued excitatory input,
+#   occurs when action potentials are triggered, causing rapid current flow in
+#   the opposite direction as the cell bodies depolarize.
+# - Inhibitory neurons, when they fire, can also contribute to negative
+#   deflections by causing hyperpolarization in their target neurons.
+# - Later oscillations likely represent ongoing network activity and
+#   subthreshold membrane potential fluctuations.
+#
+# The y-axis represents dipole amplitude in nAm (nanoAmpere-meters), which is
+# the product of current flow and distance in the neural tissue.
+#
+# Stronger synaptic connections (yellow lines) generally show larger
+# amplitude responses and more pronounced features throughout the simulation.
+
+dpl_waveforms, param_values = [], []
 for data_list in simulation_results['simulated_data']:
     for data in data_list:
         dpl_smooth = data['dpl'][0].copy().smooth(window_len=30)
         dpl_waveforms.append(dpl_smooth.data['agg'])
+        param_values.append(data['param_values']['weight_basket'])
 
 plt.figure(figsize=(10, 6))
-for waveform in dpl_waveforms:
-    plt.plot(waveform, alpha=0.5, linewidth=3)
+cmap = plt.get_cmap('viridis')
+param_values = np.array(param_values)
+norm = plt.Normalize(param_values.min(), param_values.max())
```

Comment:
I would mess with the plotting here, it seems like the log scale is throwing off the colormap (most of the colors are dark)